### DISCUSSION TITLE
Pull Request Approve 하는 방법에 대한 영상입니다.

### DISCUSSION THREAD

**Discussion by Heongilee** [Heongilee/projectTeam#52] [General]
감사합니다. 🙇 

https://user-images.githubusercontent.com/55871242/129161034-c299011f-5b6a-43b1-9348-2db393d30cb6.mp4

**Comment by jungyr24** [comment]
오케 ~~ 👌🏻👌🏻 최고최고 👍🏻👍🏻👍🏻

**Reply by JinB7** [reply]
선추천 후감상
